Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -196,7 +193,26 @@ public IncrementalLoadingCollection(TSource source, int itemsPerPage = 20, Actio
         /// An object of the <see cref="LoadMoreItemsAsync(uint)"/> that specifies how many items have been actually retrieved.
         /// </returns>
         public IAsyncOperation<LoadMoreItemsResult> LoadMoreItemsAsync(uint count)
-            => AsyncInfo.Run((c) => LoadMoreItemsAsync(count, c));
+            => LoadMoreItemsAsync(count, new CancellationToken(false)).AsAsyncOperation();
+
+        /// <summary>
+        /// Clears the collection and reloads data from the source
+        /// </summary>
+        /// <returns>This method does not return a result</returns>
+        public async Task RefreshAsync()
+        {
+            if (this.IsLoading)
```

Comment:
Done. Please allow me for some time to do more testing before closing the PR. I'm using this in an actual app :)